mappings/net/minecraft/util/math/Vec3i.mapping

```diff
@@ -23,78 +37,149 @@ CLASS net/minecraft/class_2382 net/minecraft/util/math/Vec3i
 		ARG 1 vec
 	METHOD method_10260 getZ ()I
 	METHOD method_10262 getSquaredDistance (Lnet/minecraft/class_2382;)D
+		COMMENT {@return the squared distance between here (center) and {@code vec}}
+		COMMENT
+		COMMENT @see #getSquaredDistance(Vec3i, boolean)
 		ARG 1 vec
 	METHOD method_10263 getX ()I
 	METHOD method_10264 getY ()I
 	METHOD method_10268 getSquaredDistance (DDDZ)D
+		COMMENT {@return the squared distance between here (center or vertex) and {@code (x, y, z)}}
+		COMMENT If {@code center} is {@code true}, this is equivalent to {@link Vec3d#ofCenter(Vec3i)
+		COMMENT Vec3d.ofCenter(this).sqauredDistanceTo(x, y, z)}; otherwise,
+		COMMENT this is equivalent to {@code Vec3d.of(this).sqauredDistanceTo(x, y, z)}.
```

Comment:
```suggestion
		COMMENT Vec3d.ofCenter(this).squaredDistanceTo(x, y, z)}; otherwise,
		COMMENT this is equivalent to {@code Vec3d.of(this).squaredDistanceTo(x, y, z)}.
```